arelle/BetaFeatures.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+BETA_OBJECT_MODEL_FEATURE = "betaObjectModel"
```

Comment:
This branch is 130 commits behind master and recommits (as new commits) at least some (hard to tell if it's everything) of the files changed in those 130 commits. As it stands this is very difficult to review and merging as is would compromise the git history.

We'll need to rebase this branch before merging.